maintainers guide (#1)
jakelishman/qiskit: docs/maintainers_guide.rst

#################
Maintainers Guide
#################

> This document calls *mainteiner* a contributor with merging privilages.

Reviewer: ```suggestion
This document defines a *maintainer* as a contributor with merge privileges.
```

> This document defines a *maintainer* as a contributor with merging privileges.

Reviewer: ```suggestion
This document defines a *maintainer* as a contributor with merge privileges.
```

> The task described here are mostly related with the Qiskit release process and with internal processes.

Author: ```suggestion
The tasks described here are mostly related with the Qiskit release process and with internal processes.
```

> The tasks described here are mostly related with the Qiskit release process and with internal processes.

Reviewer: ```suggestion
The information detailed here is mostly related to Qiskit releases and other internal processes.
```
